pkg/ingester/tailer.go

```diff
@@ -4,9 +4,10 @@ import (
 	"encoding/binary"
 	"hash/fnv"
 	"sync"
-	"sync/atomic"
 	"time"
 
+	"go.uber.org/atomic"
```

Comment:
was this relevant to fixing the panic?

Reply:
not really, it was fixing faillint from this build. https://github.com/grafana/loki/actions/runs/9107236736/job/25035770691